### PR TITLE
Add json files for 2016 MuonEG dataset

### PR DESCRIPTION
**Pull request by Andrew42**
This PR adds the json files for the MuonEG dataset that were missing from the 2016 data samples.

I've only included the generated json files in this PR, without updating `make_jsons.py` and also without updating the corresponding `cfg` files. This is because this dataset corresponds to the NAOD-v9 version, which is being handled more generally in PR #146, which already includes the naming changes for those files (but will still need to be updated to include this missing dataset).

I think it will be easier to include just the missing MuonEG json files here and then have these changes merged back into the #146 afterwards. This should hopefully minimize any confusing merge conflicts that might show up between the two PRs.

As already mentioned, this PR introduces the dataset files corresponding to _NAOD-v9_. It should be noted that the current NAOD data samples on `master` are for NAOD-v8 (at least until PR #146 is merged). This means that we probably shouldn't actually include this dataset in any CR studies/plots until after #146 is finally merged.

### CODE REVIEW

**Comment by kmohrman** [comment]
This looks like a straightforward PR to merge. I agree with your proposal for how to proceed (i.e. merging this PR, but not using it till #146 is merged). I'd be happy to merge this PR unless anyone else has any objections.

**Review by bryates — APPROVED** [review]
Looks good to me. We can merge now and not use it until #146 is merged, or just fold this into #146. I'm fine with whatever is easier.

**Pull request merged**
